sql/python/sqlflow_submitter/db_test.py

```diff
@@ -94,8 +94,10 @@ def _do_test(self, driver, conn):
             execute(driver, conn, self.hive_create_statement)
         else:
             execute(driver, conn, self.create_statement)
-
-        insert_values(driver, conn, table_name, table_schema, values)
+        writer = db_writer_factory(driver, conn, table_name, table_schema)
+        for row in values:
+            writer.write(row)
+        writer.close()
```

Comment:
Wrap this in a with block or finally block to make sure it’s executed even when the write() call fails

Reply:
Good suggestion! Updated PR based on your comment.